exchanges/okx/okx.go

```diff
@@ -1327,13 +1142,13 @@ func (ok *Okx) GetQuotes(ctx context.Context, arg *QuoteRequestParams) ([]QuoteR
 		params.Set("limit", strconv.FormatInt(arg.Limit, 10))
 	}
 	var resp []QuoteResponse
-	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getQuotesEPL, http.MethodGet, common.EncodeURLValues(rfqQuotes, params), nil, &resp, true)
+	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getQuotesEPL, http.MethodGet, common.EncodeURLValues("rfq/quotes", params), nil, &resp, request.AuthenticatedRequest)
 }
 
-// GetRfqTrades retrieves the executed trades that the user is a counterparty to (either as the creator or the receiver).
+// GetRfqTrades retrieves executed trades where the user is a counterparty, either as the creator or the receiver
 func (ok *Okx) GetRfqTrades(ctx context.Context, arg *RfqTradesRequestParams) ([]RfqTradeResponse, error) {
```

Comment:
```suggestion
func (ok *Okx) GetRFQTrades(ctx context.Context, arg *RfqTradesRequestParams) ([]RfqTradeResponse, error) {
```
Will stop reporting these and you've already adjusted a lot of them, but please update all references in line with https://google.github.io/styleguide/go/decisions.html#initialisms